py3status/modules/hueshift.py

```diff
@@ -0,0 +1,202 @@
+# -*- coding: utf-8 -*-
+"""
+Shift the color temperature of your screen.
+
+Configuration parameters:
+    button_down: mouse button to change color temperature (default 5)
+    button_reset: mouse button to reset color temperature (default 3)
+    button_up: mouse button to change color temperature (default 4)
+    command: specify sct, blueshift or redshift to use, otherwise automatic
+        (default None)
+    delta: specify interval value to change color temperature (default 250)
+    format: display format for this module
+        (default '[\?color=degraded \[{pgrep}\]|[\?color=hue&show \u263c]
+        [\?if=is_scrolling {command} {color_temperature}K]]')
+    initial: specify initial color temperature to use (default 6500)
+    maximum: specify maximum color temperature to use (default 25000)
+    minimum: specify minimum color temperature to use (default 1000)
+    scroll_timeout: specify number of milliseconds to use (default 5)
+    thresholds: specify color thresholds to use
+        (default [(6499, '#f3c'), (6500, '#ff3'), (6501, '#3cf')])
+
+Control placeholder:
+    {is_scrolling} a boolean based on mouse scrolling data
+
+Format placeholder:
+    {color_temperature} color temperature, eg 6500K
+    {command} command name, eg sct, blueshift, redshift
+    {pgrep} pgrep name, eg sct, blueshift, redshift
+
+Color options:
+    color_degraded: paused
+
+Color thresholds:
+    hue: print color based on value of color temperature
+
+Requires:
+    sct: set color temperature
+    blueshift: an extensible and highly configurable alternative
+    redshift: program to adjust the color temperature of your screen
+
+Suggestions:
+    campfire: 4500
+    dust storm on mars: 2000
+    coffee free all nighter: 8000
+
+@author lasers
+
+SAMPLE OUTPUT
+# warm
+[
+    {'full_text': u'\u263c', 'color': #f3c},
+    {'full_text': u'redshift 3500K'}
+]
+
+# neutral
+{'full_text': u'\u263c', 'color': #ff3}
+{'full_text': u'sct 3500K'}
+
+# cool
+[
+    {'full_text': u'\u263c', 'color': #3cf},
+    {'full_text': u'blueshift 10000K'}
+]
+
+paused
+{'full_text': u'\[sct, blueshift, redshift\]', 'color': #ff0}
+"""
+
+STRING_NOT_INSTALLED = 'not installed'
+STRING_BAD_COMMAND = 'invalid command'
+
+
+class Py3status:
+    """
+    """
+    # available configuration parameters
+    button_down = 5
+    button_reset = 3
+    button_up = 4
+    scroll_timeout = 5
+    command = None
+    delta = 250
+    format = u'[\?color=degraded \[{pgrep}\]|[\?color=hue&show \u263c]' +\
+        '[\?if=is_scrolling {command} {color_temperature}K]]'
+    initial = 6500
+    maximum = 25000
+    minimum = 1000
+    thresholds = [(6499, '#f3c'), (6500, '#ff3'), (6501, '#3cf')]
+
+    def post_config_hook(self):
+        hueshift_commands = ['sct', 'blueshift', 'redshift']
```

Comment:
I'd make `hueshift_commands` a constant ie `HUESHIFT_COMMANDS` for better readability

Reply:
This only occurs in `post_config_hook` though? Maybe I make it more vague? `cmds` ?